Fulp T5 Trekkie Chems and Comebacks Rework

Saliferous opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Attempts to fix several issues pertaining to tg's chem updates
Brings back Trekkie chems in a more balanced fashion along with adapting upon the T5 upgrade parts patch. Includes other throwback chems.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->
Adds new files: fulp_chem_CF_reagents.dm , fulp_chem_CF_medicines.dm
Edits file: chem_dispenser.dm to allow for an upgraded chem list past T4, pill.dm for charcoal add-on, cat2_medicine_reagents.dm for sanguibital desc edit
Adds reagents for Trekkie Chems and some others that allow chemistry to operate efficiently
NOTICE: Requires Xeon's T5 Upgrade Parts
The 'new' Trekkie Chems are Inject-Only, to limit the swarm of applications and to coerce players to use something other than patches or pills.
Also adds Charcoal back in but it acts as good-ole' charcoal instead of Multiver. 
Additionally, code for a replacement Synthflesh and Perfluorodecalin is included but commented out to prevent dupes. This is added for the intent to replace those chems once tg decides to wipe them out as well.
## Why It's Good For The Game
Tries to quite literally 'un-fuck' the catastrophe that is tg's latest update to chemistry by providing better endgame chem options with added T5 upgrade parts, will allow chemistry to streamline medical chems so they can focus on other objectives, and adding back in crucial chems that allow chemistry to function throughout the round. Also acts as a way of applying high-level research in another aspect, as an advanced society would have advanced medical chems. Just gotta use those hyposprays, Scotty!
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: New Files: fulp_chem_CF_reagents , fulp_chem_CF_medicines
add: New reagents (Trekkie Chems: Bicaridine, Kelotane, AntiToxin, Tricordrazine)
add: New reagent (Trekamol: Activator required for Trekkie Chems)
add: New reagent (Charcoal: Adds it back in as regular charcoal, acts similar to multiver without the bull)
add: commented out reagents in fulp_chem_CF_reagents.dm
edit: pill.dm
edit: cat2_medicine_reagents.dm
edit: chem_dispenser.dm
edit: tgstation.dme
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Saliferous committed:
Sanobital Desc Edit

Who tf would write "whatever vampires would eat" as a description for a taste, thats dumb af

Saliferous committed:
Chem Rework

Adds trek chems back in as well as charcoal, synthflesh, and perfluoro.
Includes balancing.

Saliferous committed:
Chem Rework rework

Saliferous committed:
enviro push

Saliferous committed:
chem rework rework rework

Saliferous committed:
Chem rework

Saliferous committed:
Chem rework

Saliferous committed:
chem rework

Saliferous committed:
chem rework too many tabs r bad

Saliferous committed:
chem rework REM is fun

Saliferous committed:
chem rework bicar test

Saliferous committed:
chem rework bicar stuff test thing

Saliferous committed:
bicar chem test

Saliferous committed:
chem rework

Saliferous committed:
do you even read these?

Saliferous committed:
chem stuff code words yatta yatta

Saliferous committed:
chem rework bicar loop test

Saliferous committed:
bicar test loop stuff thing whatever

Saliferous committed:
bad bicar stuff test loop

Saliferous committed:
Chem Rework (Added chem loops to have my cake and eat it too)

Saliferous committed:
Chem rework Loop final with T5 comments added (will change)

Saliferous committed:
T5 chem dispenser patch

Saliferous committed:
Chem rework T5/trekchem

Saliferous committed:
I can't spell medicine apparently

Saliferous committed:
More //FULP comments in non-fulp code

CosmicPhoenix commented:
You need to add back Styptic Powder and Silver Sulf. They don't exist anymore.

TheSwain commented:
Did they really remove styptic powder? What's the replacement now? How do you handle brute damage?

So all that's left I think is to move the item code out of code/modules/reagents/reagent_containers/pill.dm and into a custom .dm file, I think, and we'll be good to try this one out live.

CosmicPhoenix commented:
Styptic Powder was replaced with Libital and the now replaced Sanguibital. Sanguibital was replaced with something that scales with your other damages so MDs will choke out their patients to heal faster with it.

LLIkolnik commented on code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm:
```diff
@@ -9,7 +9,7 @@
 	description = "A unique medicine that heals bruises, scaling with the rate at which one is bleeding out. Dilates blood streams, increasing the amount of blood lost. Overdosing further increases blood loss."
 	color = "#ECEC8D" // rgb: 236	236	141
-	taste_description = "whatever vampires would eat"
+	taste_description = "tastes like copper"
```
"tastes" shouldn't be here.

Saliferous committed:
Additional Patchwork/Item Moving/Desc Renaming/etc

Moved the charcoal item from pill.dm to new file fulp_chem_CF_reagent_containers.
Edited the Sanobital taste desc to just 'copper'.
Fixed issues with TrekChem OD (May need more adjustment).
Ready for Live Testing Baby!

Saliferous committed:
fixed T3 and up upgrades for swain

Saliferous committed:
more swain stuff

TheSwain committed:
Update tgstation.dme
